tests/frontend/localized_time_controller_test.js

```diff
@@ -39,7 +39,7 @@ describe("Localized time controller", () => {
       expect(el).toHaveTextContent("Sep 20, 2019");
       // The expected ISO string in the title is localized
       const date = new Date(el.getAttribute("datetime"));
-      const expectedDate = format(date, "YYYY-MM-DD HH:mm:ss");
+      const expectedDate = format(date, "YYYY-MM-DD HH:mm:ss (Z)");
```

Comment:
Hm, is `Z` excpected to be the timezone here? Could we setup the test browser to be based somewhere in particular ?

Reply:
It's called [`libfaketime`](https://github.com/wolfcw/libfaketime). We need to preload it before starting the browser and to set the correct environment variables. @di Would this be an appropriate solution to the test failure?